lib/rules/no-else-return.js

```diff
@@ -9,8 +9,10 @@
 // Requirements
 //------------------------------------------------------------------------------
 
+const _ = require("lodash/fp");
 const astUtils = require("../ast-utils");
 const FixTracker = require("../util/fix-tracker");
+const getAllowElseIf = _.getOr(true, "options[0].allowElseIf");
```

Comment:
Personally, I think this is an unnecessary level of indirection -- the meaning isn't clear unless the reader has read the `lodash/fp` documentation. In my opinion, something like this would be better:

```js
const allowElseIf = !(context.options[0] && context.options[0].allowElseIf === false);
```

Reply:
lodash is pretty elegant, and already included in eslint...